include/libdnf/utils/convert.hpp

```diff
@@ -17,29 +17,19 @@ You should have received a copy of the GNU Lesser General Public License
 along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-
-#ifndef LIBDNF_UTILS_UTILS_INTERNAL_HPP
-#define LIBDNF_UTILS_UTILS_INTERNAL_HPP
-
+#ifndef LIBDNF_UTILS_CONVERT_HPP
+#define LIBDNF_UTILS_CONVERT_HPP
 
 #include <ctype.h>
 
-#include <cstring>
 #include <string>
 
-
 namespace libdnf::utils {
 
-inline bool is_glob_pattern(const char * pattern) {
-    return strpbrk(pattern, "*[?") != nullptr;
-}
-
-/// @brief Test if pattern is file path
-/// Return true if pattern start with "/" or pattern[0] == '*' && pattern[1] == '/'
-static inline bool is_file_pattern(const std::string & pattern) {
-    return pattern[0] == '/' || (pattern[0] == '*' && pattern[1] == '/');
-}
-
+/// @brief Convert an input text to a lowercase version.
```

Comment:
I am really sorry, but I don't understand why the file was renamed and moved to the public part? It looks like that it is used by package_query.cpp or did I overlooked something?

Reply:
Basically I wanted to move the `is_file_pattern` and other functions that have the global meaning to the API. Here it was in the `utils_internal.hpp` which by the name tells it is not the API. So I moved the functions and tried to group them to files by their purpose - `convert.hpp` and `patterns.hpp`

Reply:
I've updated the docstring in the `context.hpp`. But anyway for me it seems this is not the right place for this function. It is sticked to the context file, but is static and related only to the argument parser...

Reply:
I am still not sure here whether I understand it correctly. I propose to make move `convert.hpp` to private part -> `libdnf/utils/convert.hpp`, because I don't see any usage in public space. Or did I overlooked something?

Reply:
Please check the end of files, there is a missing newline and I believe that it is part of the standard.

Reply:
I see now what was the main concern about. I moved the `convert.hpp` into the private part as you're right there are no API usages (yet). And sorry for the confusion I probably somehow thought you are addressing the `is_file_pattern` function in the original comment.